app/serializers/sprangular/payment_source_serializer.rb

```diff
@@ -10,5 +10,35 @@ def include_gateway_payment_profile_id?
     def include_gateway_customer_profile_id?
       scope.has_spree_role?("admin")
     end
+
+    def month
+      object.respond_to?(:month) ? object.month : ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.